components/core/src/clp/clo/CommandLineArguments.hpp

```diff
@@ -45,6 +51,9 @@ class CommandLineArguments : public CommandLineArgumentsBase {
     // Variables
     std::string m_search_controller_host;
     std::string m_search_controller_port;
+    std::string m_mongodb_uri;
+    std::string m_mongodb_database;
```

Comment:
Can we make the user specify this as part of the URI (so we only need to support fewer parameters)?

Reply:
But we need to parse it in the source code (as far as I know there's no method that returns mongocxx::database)?

Reply:
There is no? https://mongocxx.org/api/mongocxx-3.8.0/classmongocxx_1_1uri.html#a3e7a589aa154fd1a6590a6056cddf602

Reply:
I mean mongocxx::database, but yeah std::string is ok